lua/nvim-navbuddy/actions.lua

```diff
@@ -566,6 +566,36 @@ function actions.toggle_preview()
 	}
 end
 
+function actions.vsplit()
+  local callback = function(display)
+    actions.close().callback(display)
+    vim.api.nvim_command("vsplit")
+    display.for_win = vim.api.nvim_get_current_win()
+    actions.select().callback(display)
+    vim.api.nvim_command("normal! zv")
+  end
+
+  return {
+    callback = callback,
+    description = "Open selected node in a vertical split"
+  }
+end
+
+function actions.hsplit()
+  local callback = function(display)
+    actions.close().callback(display)
+    vim.api.nvim_command("split")
+    display.for_win = vim.api.nvim_get_current_win()
+    actions.select().callback(display)
+    vim.api.nvim_command("normal! zv")
+  end
+
+  return {
+    callback = callback,
+    description = "Open selected node in a horizontal split"
+  }
+end
+
```

Comment:
Use tabs for indentation

Reply:
Done, would you consider adding `.editorconfig`(neovim has [native support](https://neovim.io/doc/user/editorconfig.html) for it), so that indentation style is always consistent among contributors?

Reply:
Yep, will look into adding that